docs/standard/commandline/syntax.md

```diff
@@ -0,0 +1,577 @@
+
+* `-v` for `--verbosity`.
+
+  Commands often provide output to the user on the console; this option is used to specify the amount of output the user requests. For more information, see [The `--verbosity` option](#the---verbosity-option) later in this article.
+
+There are also some aliases with common usage limited to the .NET CLI. You can use these aliases for other options in your apps, but be aware of the possibility of confusion.
+
+* `-c` for `--configuration`
+
+  This option often refers to a named Build Configuration, like `Debug` or `Release`. You can use any name you want for a configuration, but most tools are expecting one of those. This setting is often used to configure other properties in a way that makes sense for that configuration&mdash;for example, doing less code optimization when building the `Debug` configuration.
+
+* `-f` for `--framework`
+
+  This option is used to select a single [Target Framework Moniker (TFM)](../frameworks.md) to execute for, so if your CLI application has differing behavior based on which TFM is chosen, you should support this flag.
+
+* `-p` for `--property`
+
+  If your application eventually invokes MSBuild, the user is going to need to customize that call in some way. This option allows for MSBuild properties to be provided on the command line and passed on to the underlying MSBuild call. If your app doesn't use MSBuild but needs a set of key-value pairs, consider using this same option name to take advantage of users' expectations.
+
+* `-r` for `--runtime`
+
+  If your application can run on different runtimes, or has runtime-specific logic, consider supporting this option as a way of specifying a [Runtime Identifier](../../core/rid-catalog.md). If your app supports --runtime, consider supporting `--os` and `--arch` also. These options let you specify just the OS or the architecture parts of the RID, leaving the part not specified to be determined from the current platform. For more information, see d[dotnet publish](../../core/tools/dotnet-publish.md).
+
+### Short names
+
+Make command names as short and easy to spell as possible. For example, if `class` is clear enough don't make the command `classification`.
+
+### Lowercase names
+
+Define names in lowercase only, except you can make uppercase aliases to make commands or options case insensitive.
+
+### Kebab case names
+
+Use [kebab case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) to distinguish words. For example, the .NET CLI option that is defined as [`--additionalprobingpath`](../../core/tools/dotnet.md#runtime-options) should have been defined as `--additional-probing-path`.
+
+Case sensitivity does not apply to options that are based on enums. Enum names are matched regardless of casing.
```

Comment:
I think this section is about command and option names, and this seems to be on how enum values are treated. If so. it doesn't quite seem to fit.